lib/matplotlib/mathtext.py

```diff
@@ -563,7 +563,8 @@ def get_depth(self, texstr, dpi=120, fontsize=14):
         return depth
 
 
-def math_to_image(s, filename_or_obj, prop=None, dpi=None, format=None):
+def math_to_image(s, filename_or_obj, prop=None, dpi=None, format=None,
+                  color=None):
```

Comment:
this should be keyword-only.

Reply:
Sorry, I do not get it. Should s and filename_or_obj also be keyword? Or **kwargs?

Reply:
See the last item of https://matplotlib.org/stable/devel/contributing.html#adding-new-api, and the linked post.

Reply:
Ahh, thanks! I see (I think). 

I wasn't aware that one can put the * after other keyword-arguments. Worth noting though that the text does not explicitly describe this case.